src/Compilers/CSharp/Test/Semantic/Diagnostics/IOperationTests.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.Semantic.UnitTests.Diagnostics
+{
+    public class IOperationTests : CSharpTestBase
```

Comment:
There is an IOperationTests.vb file with some tests of the proper form. I think that all C# tests of similar form are in PRs that have yet to be merged. For example, PR 11450 has some examples.